### PR TITLE
Add latest release of Vamp

### PR DESCRIPTION
**Pull request by iamtew**
This will make the latest release of Vamp 0.9.1 available in Universe.

Tested on 1.8, all build tests return OK.

### CODE REVIEW

**Comment by iamtew** [comment]
Can I have some clarification on why the repo validation check failed? Details are hidden behind a login, which I don't have...

**Comment by cruhland** [comment]
@iamtew looks like a transient failure, a dependency failed to download. I reran it.

File: repo/packages/V/vamp/1/config.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "properties": {
+    "vamp": {
```
Comment (ryadav88) [inline repo/packages/V/vamp/1/config.json]: change `vamp` to `service`

File: repo/packages/V/vamp/1/config.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      "description": "Vamp specific configuration properties",
+      "properties": {
+        "vamp-id": {
```
Comment (ryadav88) [inline repo/packages/V/vamp/1/config.json]: `vamp-id` to `name`

File: repo/packages/V/vamp/1/config.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          "default": "vamp",
+          "description": "Marathon Vamp application id.",
```
Comment (ryadav88) [inline repo/packages/V/vamp/1/config.json]: `description` to `The name of the service to display in the DC/OS dashboard.`

File: repo/packages/V/vamp/1/marathon.json.mustache [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
+    "id": "/vamp/{{vamp.vamp-id}}",
```
Comment (ryadav88) [inline repo/packages/V/vamp/1/marathon.json.mustache]: `/service/{{service.name}}`. Also change other fields here based on changes in config.json

**Comment by iamtew** [comment]
Examples documentation added under https://github.com/dcos/examples/pull/8

**Pull request merged**